components/logo/Logo.tsx

```diff
@@ -0,0 +1,20 @@
+import AussiTentIcon from "../icons/AussiLogoTent";
```

Comment:
I would try to avoid different names.
```suggestion
import AussiTentIcon from "../icons/AussiTentIcon";
```

or
```suggestion
import AussiLogoTent from "../icons/AussiLogoTent";
```